src/contracts/AllocationModule.sol

```diff
@@ -148,6 +148,23 @@ contract AllocationModule {
         );
     }
 
+    /// @dev Returns how many COW tokens are claimable at the current point in time by the given address. Tokens that
+    /// were already claimed by the user are not included in the output amount.
+    /// @param beneficiary The address that owns the claim.
+    /// @return The amount of COW that could be claimed by the beneficiary at this point in time.
+    function claimableCow(address beneficiary) external view returns (uint256) {
+        if (allocation[beneficiary].totalAmount == 0) {
```

Comment:
This is needed because `retrieveClaimedAmounts` reverts right?

Reply:
Correct! I wanted to use `try .. catch` here but it can't be used with internal functions.